Gems/AtomLyIntegration/ImguiAtom/Code/Source/DebugConsole.cpp

```diff
@@ -63,11 +60,38 @@ namespace AZ
         }
     }
 
-    ////////////////////////////////////////////////////////////////////////////////////////////////
+    void ResetTextInputField(ImGuiInputTextCallbackData* data, const AZStd::string& newText)
+    {
+        data->DeleteChars(0, data->BufTextLen);
+        data->InsertChars(0, newText.c_str());
+    }
+
+    int InputTextCallback(ImGuiInputTextCallbackData* data)
+    {
+        DebugConsole* debugConsole = static_cast<DebugConsole*>(data->UserData);
+        if (!debugConsole)
+        {
+            return 0;
+        }
+
+        switch (data->EventFlag)
```

Comment:
Does Clang not complain about a missing default case anymore?

Reply:
Nope, apparently it does not. This is code that I moved around, I didn't edit the contents of the function in any way, so it's been functional like this for some time.